src/components/ThirdPartyCollectionDetailPage/ThirdPartyCollectionDetailPage.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react'
+import CopyToClipboard from 'react-copy-to-clipboard'
+import { Section, Row, Narrow, Column, Header, Icon, Button } from 'decentraland-ui'
+import { t, T } from 'decentraland-dapps/dist/modules/translation/utils'
+import { locations } from 'routing/locations'
+import { canSeeCollection } from 'modules/collection/utils'
+import LoggedInDetailPage from 'components/LoggedInDetailPage'
+import Notice from 'components/Notice'
+import NotFound from 'components/NotFound'
+import BuilderIcon from 'components/Icon'
+import Back from 'components/Back'
+import CollectionContextMenu from './CollectionContextMenu'
+import CollectionPublishButton from './CollectionPublishButton'
+import { Props } from './ThirdPartyCollectionDetailPage.types'
+
+import './ThirdPartyCollectionDetailPage.css'
+
+const STORAGE_KEY = 'dcl-third-party-collection-notice'
+
+export default class ThirdPartyCollectionDetailPage extends React.PureComponent<Props> {
+  handleEditName = () => {
+    const { collection, onOpenModal } = this.props
+    if (collection && !collection.isPublished) {
+      onOpenModal('EditCollectionNameModal', { collection })
+    }
+  }
+
+  handleBuySlot = () => {
+    // onOpenModal('BuySlotModal', {})
+  }
+
+  handleGoBack = () => {
+    this.props.onNavigate(locations.collections())
+  }
+
+  hasItems() {
+    const { items } = this.props
+    return items.length > 0
+  }
+
+  hasAccess() {
+    const { wallet, collection } = this.props
+    return collection !== null && canSeeCollection(collection, wallet.address)
+  }
+
+  renderPage() {
+    const { wallet, items } = this.props
+    const collection = this.props.collection!
+    const slots = 0
+
+    return (
+      <>
+        <Section className={collection.isPublished ? 'is-published' : ''}>
```

Comment:
Third party collections shouldn't ever be published, so this class won't be used.